Makefile

```diff
@@ -242,4 +242,12 @@ create-cert: stage
 	    -keyout stage/certs/aem.key \
 	    -out stage/certs/aem.cert
 
+# Creates self-signed certificate (AEM 6.5 / JDK11)
+create-cert65: stage
+	openssl genrsa -aes256 -out localhostprivate.key 4096
```

Comment:
Please move all output file to be under `stage/certs` .
That way the generated files will be cleaned up as part of stage.